src/components/missionBlock/MissionBlock.tsx

```diff
@@ -0,0 +1,114 @@
+import { ProfileProgressBar } from '../profileProgressBar/ProfileProgressBar';
+import { TimeRemaining } from '../timeRemaining/TimeRemaining';
+import './missionBlock.scss';
+
+
+export enum MissionStatusEnum {
+  EN_CURSO = 'En Curso',
+  FINALIZADA = 'Finalizada',
+}
+
+
+// TO DO: 
+// export interface Task{
+//   taskName: string;
+//   group: string;
+//   deadline: Date;
+//   status: MissionStatusEnum;
+//   percentageCompleted: number;
+// }
+
+export interface MissionBlockProps {
+  // task : Task;
+  taskName: string;
+  group: string;
+  deadline: Date;
+  status: MissionStatusEnum;
+  percentageCompleted: number;
+  backgroundColor?: string;
```

Comment:
Los estilos deberían ir por separado

Reply:
justo es lo que queda en el TO DO, pero hasta que no nos pongamos de acuerdo en definir ciertos objetos no quiero cambiarlo para no trabajar el doble :C

Reply:
¡Perfe!